src/uct/api/uct.h

```diff
@@ -1175,6 +1230,38 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief Initiate a client-server connection to a remote peer.
+ *
+ * requires @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability.
```

Comment:
It does look funky. 

I would use doxygen \warning or \note for "requires @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability."  Please put it after the function description. Also, please change it to a complete sentence. Thanks !